src/exposurelog/routers/delete_message.py

```diff
@@ -2,6 +2,8 @@
 
 __all__ = ["delete_message"]
 
+import http
```

Comment:
This is why I love doing code reviews. I didn't know about the `http` library and the status code enums are excellent